cmd/osv-scanner/main.go

```diff
@@ -99,14 +99,25 @@ func run(args []string, stdout, stderr io.Writer) int {
 			},
 			&cli.BoolFlag{
 				Name:  "experimental-call-analysis",
-				Usage: "attempt call analysis on code to detect only active vulnerabilities",
+				Usage: "[Deprecated] attempt call analysis on code to detect only active vulnerabilities",
```

Comment:
Can we just remove this flag? We make no guarantees around the stability of experimental flags anyway.

Reply:
I think Gareth made a good point that it'll be nice to give at least one release gap before deleting something for people to switch to the new flag while being able to update to the latest smoothly. We can remove it after next release without increasing complexity much.

Reply:
Makes sense!